autoextract_spiders/spiders/util.py

```diff
@@ -94,6 +94,8 @@ def maybe_is_article(url: str) -> bool:
     """
     if not could_be_content_page(url):
         return False
+    if re.search('/contact-?(us)?$', url):
```

Comment:
Hey! Just to clarify: was it intentional that about-us pages are now sent for article extraction?

Reply:
by the way, docstrings for these maybe_... functions don't make sense :)

Reply:
Yes, the "contact-us" makes sense to be removed from the articles. The "about-us" still could be an article, some blogs create a normal page with title, date and author in the about page.
You're right about the docstrings, updating now ;)